hive-event-listeners/apiary-gluesync-listener/src/main/java/com/expediagroup/apiary/extensions/gluesync/listener/ApiaryGlueSync.java

```diff
@@ -15,18 +15,11 @@
  */
 package com.expediagroup.apiary.extensions.gluesync.listener;
 
-import java.util.ArrayList;
```

Comment:
Many code changes, but esentially just moving code to classes to make it more readable